stories/index.js

```diff
@@ -430,4 +432,44 @@ storiesOf('LineChart', module)
       }
       return <DynamicBaseDomain />;
     })
+  )
+  .add(
+    'Enable/disable series',
+    withInfo()(() => {
+      const options = [
+        { value: '1', label: 'steelblue' },
+        { value: '2', label: 'maroon' },
+        { value: '3', label: 'red' },
+      ];
+      // eslint-disable-next-line
+      class EnableDisableSeries extends React.Component {
+        state = {
+          series: options,
```

Comment:
if instead of options specify array of a single series and try to add data through combobox, an error occurs:
`Uncaught TypeError: Array.prototype.map called on null or undefined`